client/signup/steps/design-picker/style.scss

```diff
@@ -125,3 +125,24 @@
 		box-sizing: border-box;
 	}
 }
+
+.design-picker__categories-footer {
+	margin-bottom: 40px;
+
+	&-button {
+		width: 100%;
+		height: 40px;
+		border-radius: 4px;
+		box-shadow: 0 1px 2px rgb( 0 0 0 / 5% );
+		font-size: 0.875rem !important; /* Overwrite style at L46 */
```

Comment:
Is it possible to use the `design-picker__categories-footer-button` class in some way to override this font style instead?

Reply:
As the value to overwrite is `inherit`, I move the `font-size` to the parent style